server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -92,10 +100,6 @@ async function updateExistingSession(
             // Reset logOffset, ordererUrl, and historianUrl when moving session location.
             Lumberjack.info("Moving session", {
                 ...lumberjackProperties,
```

Comment:
Any reason to not keep the properties like `isSessionSticky` and `documentLastAccessTime` here as well? Just in case it might make it easier to correlate with the other `"Calculated isSessionSticky and sessionHasLocation"` log added above

Reply:
Got it. Thanks for a good idea. Added it. :)